src/content/docs/resources/glossary.mdx

```diff
@@ -33,11 +33,13 @@ Wallet providers who support Open Payments issues each of their customers' accou
 
 ### Payment pointer
 
-A payment pointer is a secure, unique identifier that's assigned to payment accounts that use the Interledger Simple Payment Setup Protocol (SPSP) to exchange payment information. The shorthand for a payment pointer starts with `$`. For example, `$wallet.example.com/alice`.
+A payment pointer is similar to a wallet address. Where a wallet address begins with `https://`, a payment pointer begins with `$`. For example, `$wallet.example.com/alice`. Most payment pointers can be converted to wallet addresses by simply replacing the `$` with `https://`. For example, `https://wallet.example.com/alice`.
 
-As a consumer of web monetized content, you'll enter your payment pointer into your Web Monetization extension to link the extension to your wallet.
+Whether a wallet provider chooses to call their payment addresses *wallet addresses* or *payment pointers* is up to the provider, so you should become familiar with both terms.
 
-To web monetize your own content, you must convert your payment pointer to its URL equivalent as the monetization `<link>` element doesn't support the `$` in the `href` value. Enter your payment pointer into the field provided on <LinkOut href="https://paymentpointers.org">paymentpointers.org</LinkOut> to get your URL.
+If you're setting up the ILF's Web Monetization extension, you can enter either a wallet address (`https://`)or a payment pointer (`$`). 
```

Comment:
We should avoid "ILF" in public I think. In this particular case, skipping it makes sense too. Can add a link to the extension as well.

```suggestion
If you're setting up the Web Monetization extension, you can enter either a wallet address (`https://`)or a payment pointer (`$`). 
```

Reply:
I'm not so sure about this one. The only extension that exists at the moment is from the ILF, and I want people to know that the extension is from a trustworthy source. Also, someone else could make an extension that doesn't support the payment pointer format. 

After there's more extensions than just the ILF's, there are other places in the docs that would need to be updated slightly as well.

Reply:
Oh, I gotcha. Yeah, let me look for other instances of that.